tests/acceptance/testall

```diff
@@ -929,8 +929,15 @@ do
     esac
 done
 
-# check specified binaries, but only check RPMVERCMP on systems with rpm command
-if [ -n "$AGENT" -o -n "$CF_PROMISES" -o -n "$CF_SERVERD" -o -n "$CF_EXECD" -o -n "$CF_KEY" -o -n "$CF_SECRET" -o -n "$CF_NET" -o -n "$CF_CHECK" -o -n "$CF_RUNAGENT" -o \( -n "$(command -v rpm)" -a -n "$RPMVERCMP" \) -o -n "$DIFF" ]
+# check if rpm is used on this host, sometimes it can be installed but not used say on Ubuntu
+if [ -n "$(command -v rpm)" -a "0" != "$(rpm -qa | wc -l)" ]
```

Comment:
Probably fine assumption that non rpm native systems don't have rpm packages installed, but might be more straightforward to base it on actual OSes by looking at stuff like /etc/os-release. for another refactor ...

Reply:
Yeah, I thought of that but thought it was too complex for here and duplication of other bits elsewhere. I'd want to check for redhat/centos/suse/aix/what else?

Reply:
TODO maybe